src/model/backup.ts

```diff
@@ -39,6 +42,9 @@ interface BackupUserData {
   wallets: AllRainbowWallets;
 }
 
+const empty = () => {};
```

Comment:
We usually call those `NOOP` (as in 'no operation).

Reply:
didn't notice that before, thanks